src/test/java/com/st/utopia/cancellationservice/dao/TicketDaoTest.java

```diff
@@ -0,0 +1,53 @@
+package com.st.utopia.cancellationservice.dao;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.st.utopia.cancellationservice.model.Ticket;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TicketDaoTest {
+	
+	@Autowired
+    private TicketDao ticketdao;
+
+    @Test
+    public void createTicket() {
+    	LocalDateTime dout = LocalDateTime.parse("28-01-2019 12:00",DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm"));
+    	Ticket t = new Ticket(1 , 11, 'B',11, 1, 1500, dout,"jhytrfgr");
+    	ticketdao.save(t);
+    	assertTrue(t.getBookingid().equals("jhytrfgr"));
+    }
+
+    @Test
+    public void getTicket() {
+    	Ticket t = ticketdao.getTicket(1, 1, 'A'); 
+    	assertTrue(t.getPrice() == 123);

```

Comment:
How can this test (or any of these tests) pass? You aren't running against a live database, or assuming that tests will run in some particular order, are you?